test/parallel/test-eventtarget-once-twice.js

```diff
@@ -1,14 +1,25 @@
 'use strict';
 const common = require('../common');
-const { once } = require('events');
+const assert = require('node:assert/strict');
```

Comment:
I don't think we need this import, is assert being used at all?

Reply:
My bad! Deleted and left the side-effectful `common` import without a binding.